tests/functionaltests/functionaltest-api.h

```diff
@@ -25,7 +52,7 @@
 #ifdef TEST
 #undef TEST
 #endif
-#define TEST(__Name1, __Name2) void __Name1##__Name2()
+#define TEST(__Name1, __Name2) TEST_METHOD(__Name1##__Name2)
```

Comment:
yeah. Talked offline about the UT plan but lets make functional tests nice. Will leave the verification macros as is for now and file a task to clean those up later.

---
In reply to: [92248037](https://github.com/Microsoft/WinObjC/pull/1523#discussion_r92248037) [](ancestors = 92248037)